src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -12012,6 +12012,52 @@ public static int Main()
                 new ErrorDescription[] { new ErrorDescription { Code = (int)ErrorCode.ERR_PartialMethodToDelegate, Line = 11, Column = 38 } });
         }
 
+        [WorkItem("https://github.com/dotnet/roslyn/issues/72431")]
+        [Fact]
+        public void CS0762ERR_PartialMethodToDelegate_02()
+        {
+            var source = """
+                delegate void D();
+                partial class Program
+                {
+                    static void Main()
+                    {
+                        M1(M2<int>);
```

Comment:
Is there also a bug when using a method group like `GenericType<int>.PartialMethod`?